### DISCUSSION TITLE
executeCachedQuery() - when have you seen all matched rows?

### DISCUSSION THREAD

**Discussion by martinEchavez** [nextgenhealthcare/connect#4497] [General]
**¿How to get all the rows from a table after executing a Stored Procedure?**
`var sqlQueryClienteServidor = "EXEC [dbo].[spDatosAtencion_ClienteServidor] @Op = 'S_DatosHistoriaAdmision', @Estudio = " + Estudio;`
`var ClienteServidorQueryResultList = dbConn.executeCachedQuery(sqlQueryClienteServidor);`

Just get the values for the first row

**Comment by tonygermano** [comment]
What exactly is the problem here? You don't show your code that retrieves the data. The stored procedure is returning a ResultSet with only one row when you expect to get more than one row?
